spaceship.cpp

```diff
@@ -0,0 +1,51 @@
+#include<iostream>
+#include<windows.h>
+#include<conio.h>
+#include "spaceship.h"
+using namespace std;
+
+//DECLARING OF GOTO FUNCTION TO MOVE LOCATIONS
+void gotoxy(int x, int y)
+{
+    COORD c={x,y};
+    SetConsoleCursorPosition (GetStdHandle(STD_OUTPUT_HANDLE),c);
+}
+
+
+//CONSTRUTOR
+
+starFighter::starFighter(int _x,int _y){
+       x=_x;
+       y=_y;
+}
+
+//PAINTING SHIP
+
+void starFighter::paint(){
+
+  	gotoxy(x,y);
+  	printf("%c%c%c",62,45);// It will look something like this >-
```

Comment:
Can't we directly print it like `printf(">-");`

Reply:
Yes Susan that will work, the code was not compiled as i was stuck in between needed some help from Johan but Johan said i need to raise pull request then only he will see my code, so i had uploaded incomplete code. Anyways, thanks for pointing out errors, will try to complete the code.